e2e/test/FaultInjectionPoolAmqpTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
Please name the file <className>.<specifier>.cs (in this case specifier could be `base`?)